cmd/build/command.go

```diff
@@ -80,23 +84,26 @@ const (
 )
 
 // Build build and optionally push a Docker image
-func Build(ctx context.Context, at analyticsTrackerInterface) *cobra.Command {
+func Build(ctx context.Context, analyticsTracker analyticsTrackerInterface) *cobra.Command {
 
 	options := &types.BuildOptions{}
 	cmd := &cobra.Command{
 		Use:   "build [service...]",
 		Short: "Build and push the images defined in the 'build' section of your okteto manifest",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			options.CommandArgs = args
-			bc := NewBuildCommand(at)
+
 			// The context must be loaded before reading manifest. Otherwise,
 			// secrets will not be resolved when GetManifest is called and
 			// the manifest will load empty values.
-			if err := bc.loadContext(ctx, options); err != nil {
+			oktetoContext, err := getOktetoContext(ctx, options)
```

Comment:
NIT: Should we separate this in two different functions? one to load and one to get the context? Right now `getOktetoContext` is loading and getting the context. I think it's cleaner if we have it both separate

Reply:
now the concept *load* disappears in the command. From the `build` command POV, a function from context package returns an okteto context so I think is not bad to call `getOktetoContext` only.